test/e2e/managed_test.go

```diff
@@ -28,18 +33,15 @@ func TestManaged(t *testing.T) {
 	}
 
 	t.Logf("waiting the operator to recreate console deployment...")
-	err := wait.Poll(1*time.Second, testframework.AsyncOperationTimeout, func() (stop bool, err error) {
-		_, err = client.Deployments(consoleapi.OpenShiftConsoleOperatorNamespace).Get(consoleapi.OpenShiftConsoleName, metav1.GetOptions{})
-		if err == nil {
-			return true, nil
-		}
-		t.Logf("get deployment: %s", err)
-		if errors.IsNotFound(err) {
-			return false, nil
-		}
-		return false, err
-	})
-	if err != nil {
-		t.Fatal(err)
+
+	errChan := make(chan error)
+	go testframework.IsResourceAvailable(errChan, client, "ConfigMap")
```

Comment:
Do we want to delete each of these resources & then check each availability?

Reply:
No only the `openshift-console` deployment is deleted. Wasnt sure if we want to delete all, or just some, or only one.

Reply:
Right, only the `deployment` is deleted, but it checks if each of the other resources are still there.

Reply:
yes, thinking about it now, I probably would be better to do it the opposite way. Delete more resource (eg.Deployment, Route, ConfigMap). Then check for all the resources.